docs/concepts.md

```diff
@@ -4,39 +4,34 @@ title: Concepts
 sidebar_label: Concepts
 ---
 
-## Specification concepts
-
-Specification uses following vocabulary to define GraphQL queries
-
-
-## CRUD method
+## CRUD methods
 
 GraphQL CRUD defines different CRUD capabilities that represents 
-various operations that can be executed on the set of the objects:
+various operations that can be executed on a set of objects:
 
-- Create: creation of the object
-- Update: update of the any object properties
-- Delete: delete entire object by id 
+- Create: creation of an object
+- Update: update of any object properties
+- Delete: delete specific object by id 
 - Get: get specific object by id
-- Find: find multiple objects using query
```

Comment:
"using a query" is ambiguous... "using a filter" would be better, but I think "find multiple objects" is also sufficiently clear